Solutions/Palo Alto - XDR (Cortex)/Detection Queries/Preventive Alerts.yaml

```diff
@@ -0,0 +1,52 @@
+id: 5180e347-32fb-4a0a-9cfa-d6e0e10fc4eb
+name: Palo Alto Prevention alert
+description: |
+  'The query checks for all malicious events prevented events across all devices having LogSeverity of '6' and summarize the result.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: PaloAltoNetworksCortex
+    dataTypes:
+      - PaloAltoNetworksCortex
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - DefenseEvasion
+relevantTechniques:
+  - T1562
+query: |
+  CommonSecurityLog
+  | where DeviceVendor == "Palo Alto Networks"
+  | where DeviceProduct == "Cortex XDR"
+  | where DeviceAction == "Prevented (Blocked)"
+  | where LogSeverity =~ "6"
+  | summarize by TimeGenerated,DeviceProduct, DeviceEventClassID, LogSeverity, Activity, DeviceAction, Computer, DeviceVersion, FileHash, FileName, FilePath, RequestURL, SourceHostName, SourceUserName
+  | sort by TimeGenerated
+entityMappings:
+  - entityType: Account
+    fieldMappings:
+      - identifier: FullName
+        columnName: SourceUserName
+  - entityType: Host
+    fieldMappings:
+      - identifier: FullName
+        columnName: SourceHostName
+  - entityType: File hash
+    fieldMappings:
+      - identifier: Value
+        columnName: FileHash
+  - entityType: File
+    fieldMappings:
+      - identifier: Name
+        columnName: FileName
+  - entityType: File
+    fieldMappings:
+      - identifier: Name
+        columnName: FilePath
+   - entityType: URL
```

Comment:
There is an extra space in this line which is leading to yaml validation failure.